corecomponent/androidcomponent/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="config_navDefaultAnimTime">220</integer>
```

Comment:
Can you add this comment? (It is a little sudden) 💬 

```
<! -- referenced config_activityDefaultDur in sdk -->
```

Reply:
yes, thanks your advice! 🙏